crates/biome_module_graph/tests/snapshots/test_resolve_exports.snap

```diff
@@ -0,0 +1,375 @@
+---
+source: crates/biome_module_graph/tests/snap/mod.rs
+expression: content
+snapshot_kind: text
+---
+## /src/index.ts
+
+```ts
+/**
+ * @returns {string}
+ */
+function foo() {}
+
+export { foo, qux };
+
+/** @package */
+export function bar() {}
+
+/** @private */
+export const quz = {};
+
+/* @ignored because of incorrect amount of asterisks */
+export async function baz() {}
+
+var qux = 1;
+
+/**
+ * TODO: No types can be detected on these yet.
+ */
+export const {
+	a,
+	b,
+	c: [d, e],
+} = getObject();
+
+/**
+ * @public
+ * @returns {JSX.Element}
+ */
+export default function Component(): JSX.Element {}
+
+export * from "./reexports";
+export { ohNo as "oh\x0Ano" } from "./renamed-reexports";
+
+/**
+ * Hello, namespace 2.
+ */
+export * as renamed2 from "./renamed-reexports";
+
+```
+
+
+
+```
+Borrowed("a") => [
+  ExportOwnExport => JsOwnExport(
+    Unknown
+    Local name: a
+    JsDoc(
+      TODO: No types can be detected on these yet.
+    )
+  )
+]
+Borrowed("b") => [
+  ExportOwnExport => JsOwnExport(
+    Unknown
+    Local name: b
+    JsDoc(
+      TODO: No types can be detected on these yet.
+    )
+  )
+]
+Borrowed("bar") => [
+  ExportOwnExport => JsOwnExport(
+    Function(
+      [sync, Name: bar]
+      ReturnType(Type(Unknown))
+    )
+    Local name: bar
+    JsDoc(
+      @package
+    )
+  )
+]
+Borrowed("baz") => [
+  ExportOwnExport => JsOwnExport(
+    Function(
+      [async, Name: baz]
+      ReturnType(
+        Type(
```

Comment:
It might be nice to just drop the `Type()` wrapper completely and only print the content of it. It will remove a lot of wrapping and indentation.